meta/what_is_artsy.md

```diff
@@ -21,22 +21,32 @@ Artsy has 5 core values:
 
 We believe in bringing together the often separated worlds of art and science, emotion and logic, creativity and analysis, the subjective and the objective. While some people feel more comfortable on one side and like to keep distance from the other, we believe that magical things happen at the intersection of the two, and look to create that intersection in everything we do and in the people we hire.
 
+> **Example**: We work as engineers, but we aren't defined by that role; Artsy holds an annual [Salon](http://www.dictionary.com/browse/salon) and we encourage our engineers (and indeed, all our colleagues) to submit work. Engineer contributions to this event have included generative pixel art, interactive video game, photo, video, sculpture, painting, live music and cartographic pieces.
+
 ### People are Paramount
 
 We believe in empowering each of our team members to feel like the CEO of their area of responsibility. People are our company’s most valuable asset, and this is reflected in the high degree of responsibility and freedom we afford every team member. This approach guides how Artsy treats our people in all other realms as well.
 
+> **Example**: We hold a weekly team lunch with a healthy, delicious meal; after introducing any new colleagues, our People Ops team encourages everyone to sit and eat with people they don't normally get to talk to.
```

Comment:
I wonder if S'up would be a good example here, especially since it came out of engineering.